src/test/java/org/cactoos/iterator/ImmutableTest.java

```diff
@@ -58,7 +58,11 @@ public void decoratesNext() {
         final Iterator<Integer> immutable = new Immutable<>(
             new IteratorOf<>(value)
         );
-        MatcherAssert.assertThat(immutable.next(), new IsEqual<>(value));
+        new Assertion<>(
+            "wrong value is provided",
```

Comment:
@ccostin93 Normally in the `cactoos`, always use the format: "Must ...", followed by what the object under test must do. Can you fix it here?

Reply:
fixed